test/runtests.jl

```diff
@@ -1,36 +1,35 @@
 using StaticArrays
 using Base.Test
 
-@testset "StaticArrays" begin
-    include("SVector.jl")
-    include("MVector.jl")
-    include("SMatrix.jl")
-    include("MMatrix.jl")
-    include("SArray.jl")
-    include("MArray.jl")
-    include("FieldVector.jl")
-    include("Scalar.jl")
-    include("SUnitRange.jl")
-    include("SizedArray.jl")
-    include("custom_types.jl")
```

Comment:
With the triangular tests included the tests take a little longer than 10 minutes to run and if Travis doesn't see any output for longer than 10 minutes it terminates. @andyferris suggested removing the outer testset to prevent this.